src/main/java/org/sagebionetworks/bridge/hibernate/QueryBuilder.java

```diff
@@ -16,37 +18,73 @@
 import org.sagebionetworks.bridge.BridgeUtils;
 import org.sagebionetworks.bridge.models.studies.EnrollmentFilter;
 
-import com.google.common.base.Joiner;
-
 /**
  * A helper class to manage construction of HQL strings.
  */
 class QueryBuilder {
     
     private final List<String> phrases = new ArrayList<>();
     private final Map<String,Object> params = new HashMap<>();
+    private QueryBuilder whereClause;
+    
+    public QueryBuilder startWhere() {
+        whereClause = new QueryBuilder();
+        return whereClause;
```

Comment:
Make a WhereQueryBuilder with some of these methods moved over, just so later it doesn't get confusing.

Reply:
Done